site/docs/components/plots/bullet.zh.md

```
@@ -48,21 +85,20 @@ order: 10
-<description>**required** _BulletOptions_</description>
-
-range 配置
+### Shape
 
-### measure 
+配置对应图形样式，常见的是 `style` ，更多配置可参考通用组件[Column](/components/plots/column#配置项)
 
-<description>**required** _BulletOptions_</description>
+| 属性    | 描述                          | 类型               | 默认值     | 必选 |
+| ------- | ----------------------------- | ------------------ | ---------- | ---- |
+| style | Mark 样式     | [详见>>](options/plots/style#配置图形样式)        |  -    |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)